packages/land/contracts/Land.sol

```diff
@@ -40,17 +42,21 @@ contract Land is LandBaseToken, OperatorFiltererUpgradeable {
      * @param metaTransactionContract Authorized contract for meta-transactions
      * @param admin Admin of the contract
      * @param _royaltyManager address of the manager contract for common royalty recipient
+     * @param _newOwner address of new owner
      * @param version version number to which Land contract is being upgraded
      */
     function initialize(
         address metaTransactionContract,
         address admin,
         address _royaltyManager,
+        address _newOwner,
```

Comment:
same here, we have 

```
 address admin,
 but
address _newOwner,
```

why is it different ? What's the logic ?

Reply:
updated